apps/web/src/components/VerificationCode/SelectMethod.tsx

```diff
@@ -0,0 +1,71 @@
+import { Animate } from "@/components/Common/Animate";
+import { Typography } from "@litespace/ui/Typography";
+import { MethodButton } from "@/components/VerificationCode/MethodButton";
+import Telegram1 from "@litespace/assets/Telegram1";
+import Whatsapp from "@litespace/assets/WhatsApp";
+import { useFormatMessage } from "@litespace/ui/hooks/intl";
+import { IUser, Void } from "@litespace/types";
+import { Button } from "@litespace/ui/Button";
+
+type SelectMethodProps = {
+  activeMethod: IUser.NotificationMethodLiteral | null;
+  changeMethod: (method: IUser.NotificationMethodLiteral) => void;
+  changeStep: (step: "phone" | "code") => void;
+  close: Void;
+};
+
+export function SelectMethod({
+  activeMethod,
+  changeStep,
+  changeMethod,
```

Comment:
The `SelectMethod` component should never have the ability to change the step! 

A better design will be something like this: 


```suggestion
    select(method: Method): void;
    next(): void;
```